mods/tuxemon/l18n/fr_FR/LC_MESSAGES/base.po

```diff
@@ -1719,10 +1719,10 @@ msgid "dodge_increased"
 msgstr "{stat} de {user} augmenté."
 
 msgid "attack_resisted"
-msgstr "L'attaque a résisté !"
+msgstr "L'adversaire a résisté à l'attaque !"
```

Comment:
Thanks, I was always confused by the previous translation...

Reply:
No problem. I decided to just retranslate the sentence entirely because the formulation didn't work as well in French as in English